doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md

```diff
@@ -57,22 +57,37 @@ It must be said [PostgreSQL only accepts](https://www.postgresql.org/docs/curren
 
 PostgreSQL [databases name length](http://www.postgresql.org/docs/current/static/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS) is limited to 63 characters.
 
+Since version 2.2.0 It's added a new capability for Cygnus to create the schema name on runtime, this is possible trough enabling a specific Data Model on agent properties. That works the following way.
+
+* Data model by entity database (`data_model=dm-by-entity-database`). For this datamodel the name of the database will be auto generated by the sink, this name will be the `fiware-service` found on the headers of the first request incoming to the sink.
+* Data model by entity database schema (`data_model=dm-by-entity-database-schema`). For this datamodel the name of the database will be auto generated by the sink, this name will be the `fiware-service` found on the headers of the first request incoming to the sink.
+
 [Top](#top)
 
 #### <a name="section1.2.2"></a>PostgreSQL schemas naming conventions
 A schema named as the notified `fiware-service` header value (or, in absence of such a header, the defaulted value for the FIWARE service) is created (if not existing yet).
 
+* Data model by entity database schema (`data_model=dm-by-entity-database-schema`). For this datamodel the name of the schema will be auto generated by the sink, this name will be the `fiware-servicePath` found on the headers of the first request to store.
+
 It must be said [PostgreSQL only accepts](https://www.postgresql.org/docs/current/static/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS) alphanumeric characters and the underscore (`_`). This leads to  certain [encoding](#section2.3.4) is applied depending on the `enable_encoding` configuration parameter.
 
 PostgreSQL [schemas name length](http://www.postgresql.org/docs/current/static/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS) is limited to 63 characters.
 
+Since version 2.2.0 Cygnus creates the name of the schema name on runtime according to the selected DataModel for the sink.
+
+* Data model by entity database schema (`data_model=dm-by-entity-database-schema`). For this datamodel the name of the schema will be auto generated by the sink, this name will be the `fiware-servicePath` found on the headers of the first request to store.
```

Comment:
It seems to be duplicated in lines L70-71. Maybe L70-71 should be removed?

Reply:
Fixed in [9bcedc4](https://github.com/telefonicaid/fiware-cygnus/pull/1876/commits/9bcedc4805e6b3e4e48b7e5a357e18931cd62a0a)